### DISCUSSION TITLE
OMI glTF Working Group Meeting 2022-09-22

### DISCUSSION THREAD

**Discussion by robertlong** [omigroup/gltf-extensions#107] [General]
This meeting will be on 9/22/2022 at 10:30 PM (UTC) / 2:30 PM (PST) in the [OMI Discord](https://discord.gg/NJtT9grz5E) within the #omi-weekly-meeting channel. During the meeting, we will be using the #omi-gltf-extensions channel to manage a speaker queue, post links, and for any sidebar discussions.

To be notified of this meeting and others, subscribe to the [OMI Meetings and Events Calendar](https://calendar.google.com/calendar/u/1?cid=Y18wZHB1Z2Y5ZjgzZXE0cWVrbWI2b21xYmptZ0Bncm91cC5jYWxlbmRhci5nb29nbGUuY29t) or add yourself to the @omi-gltf-subgroup role in the #roles channel of the OMI Discord.

## Agenda:

- glTF Transform Updates
- [OMI_collider](https://github.com/omigroup/gltf-extensions/pull/63)
- [KHR_audio / Khronos feedback](https://github.com/KhronosGroup/glTF/pull/2137)
- [glXF](https://github.com/KhronosGroup/glXF/blob/main/specification/2.0/README.md)
  - OMI_id / [OMI_ref](https://github.com/omigroup/gltf-extensions/pull/85)
- USDz spike
- Schedule alternate timezone meeting
- Show and tell
- Create next weeks agenda

To propose another item for the agenda, comment below (preferably before the meeting).

**Comment by antpb** [comment]
• glXF: https://github.com/KhronosGroup/glXF/blob/main/specification/2.0/README.md

 Names are not unique across the document and currently lacks unique identifiers. For ids, how do you subtarget by id? glXF does not seem to address naming conflicts. It was noticed in the meeting that the assets can actually have duplicate names because the nodes further in the document can reference by index rather than name. Is this the desired way?
• antpb has a bit of anxiety about a new filetype given the complexity platforms with upload capabilities will face maintaining multiple file types.

•USDz - is only two widely used 3D file types too few?
• https://github.com/syoyo/tinyusdz we discovered there was webassembly support recently added and we discussed a bit of three.js recent support challenges faced with usdc files.

• VRM has reached 1.0 and has new shiny extensions based on attributes, bones, materials, etc.

• Antpb suggested moving the meeting back 30 minutes to allow his availability to lead the meeting if needed and catch all the important info that he often misses. Alternatively, we could use the first 30 minutes for discussion leading up to an official opening of the meeting 30 minutes later.
